models/issue.go

```diff
@@ -1285,22 +1285,22 @@ func UpdateIssueUserByRead(uid, issueID int64) error {
 }
 
 // UpdateIssueUsersByMentions updates issue-user pairs by mentioning.
-func UpdateIssueUsersByMentions(issueID int64, uids []int64) error {
+func UpdateIssueUsersByMentions(e Engine, issueID int64, uids []int64) error {
 	for _, uid := range uids {
```

Comment:
If you mean this operation need a transaction, we should use `Begin()` and `Commit()`.
```Go
sess := x.NewSession()
defer sess.Close()
if err := sess.Begin(); err != nil {
    return err
}

...
return sess.Commit()
```